ledger-core/src/core/wallet/WalletStore.cpp

```diff
@@ -73,6 +97,14 @@ namespace ledger {
             });
         }
 
+        void WalletStore::getWalletCount(
+            const std::function<void(std::experimental::optional<int32_t>, std::experimental::optional<api::Error>)> & callback
+        ) {
+            getWalletCount().map<int32_t>(_services->getContext(), [] (const int64_t count) {
+                return (int32_t) count;
```

Comment:
Please replace C cast by the proper C++ one.

Reply:
Good catch.